index.html

```diff
@@ -122,8 +124,6 @@ <h3>CONTACT</h3>
         <input type="text" id="message"><br>
```

Comment:
To help with styling and structure, use a footer for these if you want them to be at the bottom and wrap them in divs

Reply:
made this change